Python/pytime.c

```diff
@@ -1059,7 +1059,7 @@ _PyTime_localtime(time_t t, struct tm *tm)
     return 0;
 #else /* !MS_WINDOWS */
 
-#ifdef _AIX
+#if defined(_AIX) && (SIZEOF_TIME_T == 4)
```

Comment:
*Maybe* it would be more explicit that supporting years after 2038 requires 64-bit time_t if you write the condition like that:

```suggestion
#if defined(_AIX) && (SIZEOF_TIME_T < 8)
```